src/lib/progress-spinner/progress-spinner.spec.ts

```diff
@@ -203,6 +203,12 @@ describe('MatProgressSpinner', () => {
     expect(svgElement.getAttribute('viewBox')).toBe('0 0 38 38');
   });
 
+  it('should update the element size when changed dynamically', () => {
+    const fixture = TestBed.createComponent(MatProgressSpinner);
+    const spinner = fixture.componentInstance;
+    spinner.diameter = 32;
+    expect(spinner._elementSize).toBe(32);
```

Comment:
Can you test the styles that are applied to the element, rather than the property itself? You can look at how the other tests are doing it.